src/services/auth.services.ts

```diff
@@ -54,12 +54,19 @@ export default class AuthServices {
    */
   static async fetchUserInfo (): Promise<any> {
     const url = `${this.authApiUrl}/users/@me`
-
-    return axios.get(url)
-      .then(response => {
-        if (response?.data) return response.data
-        throw new Error('Invalid user info')
-      })
+    if (!sessionStorage.getItem('PREVENT_STORAGE_SYNC')) {
+      const token = sessionStorage.getItem('KEYCLOAK_TOKEN')
+      if (token) {
+        const headers = {
+          Authorization: `Bearer ${token}`
+        }
+        return axios.get(url, { headers: headers })
+          .then(response => {
+            if (response?.data) return response.data
+            throw new Error('Invalid user info')
+          })
+      }
+    }
```

Comment:
This function doesn't return anything if the condition on line 57 is False.

It might be cleaner to change line 57 to something like this:
`if (!sessionStorage.getItem('PREVENT_STORAGE_SYNC')) return null`

Note1: Please add a comment above line 57 explaining something about PREVENT_STORAGE_SYNC and why we're checking it here.

Note2: In my sample above, I return null, so you might want to change the check in update-user-mixin.ts to `if (!userInfo)`.